examples-shared/src/components/modals/SnappingList.tsx

```diff
@@ -48,6 +52,15 @@ export default class SnappingList extends React.PureComponent<IProps> {
     }
   }
 
+  private scrollDown = () => {
```

Comment:
Could you move this button at the bottom of the list and make it "scroll to the top" instead?